internal/dag/httpproxy_processor.go

```diff
@@ -993,8 +993,18 @@ func (p *HTTPProxyProcessor) computeRoutes(
 				return nil
 			}
 			if service.Mirror {
-				r.MirrorPolicy = &MirrorPolicy{
-					Cluster: c,
+				// Default value if weight is omitted is 0. Setting this to 100 to retain backward-compat behavior.
+				// Edge case is a user intentionally/explicitly setting the Weight to 0 (ie. disabling the mirror).
```

Comment:
I'd also clarify the comment here since the user setting the weight to 0 will _not_ disable the mirror since it's indistinguishable from the "weight not specified" case; they'd need to remove the service from the route.

Reply:
Clarified the edge case in the comment.